MapboxNavigation/PollyVoiceController.swift

```diff
@@ -57,14 +68,31 @@ public class PollyVoiceController: RouteVoiceController {
         
         pollyTask?.cancel()
         audioPlayer?.stop()
+        startAnnouncementTimer()
+        
+        for (stepIndex, step) in routeProgresss.currentLegProgress.leg.steps.suffix(from: routeProgresss.currentLegProgress.stepIndex).enumerated() {
+            let adjustedStepIndex = stepIndex + routeProgresss.currentLegProgress.stepIndex
+            
+            guard adjustedStepIndex < routeProgresss.currentLegProgress.stepIndex + 3 else { continue }
```

Comment:
Should this be using the `stepsAheadToCache` instead of 3?

Reply:
Oh yep.